inference-engine/src/vpu/graph_transformer/src/middleend/passes/mark_fast_stages.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vpu/middleend/pass_manager.hpp>
+
+namespace vpu {
+
+namespace {
+
+class PassImpl final : public Pass {
+public:
+    void run(const Model& model) override {
+        VPU_PROFILE(markFastStages);
+
+        for (const auto& stage : model->getStages()) {
+            const auto& outputs = stage->outputs();
+            const auto& itSlowStage = std::find_if(outputs.begin(), outputs.end(), [](const Data& output) {
+                return output->desc().totalDimSize() > 100;
+            });
+
+            if (itSlowStage == outputs.end()) {
```

Comment:
How do you feel about such a method?
`std::all_of(..., {output->desc().totalDimSize() <= 100})`

Reply:
It looks way better now. Thanks!